common/app/views/fragments/inlineJSNonBlocking.scala.html

```diff
@@ -30,4 +30,32 @@
 // Ophan pageview ID and browser ID are needed by Google Analytics, which runs just after this script tag
 @InlineJs(ophanConfig().body, "ophanConfig.js")
 
+@if(WebpackSwitch.isSwitchedOn) {
```

Comment:
I have created a stripped-down version of `standard/main.js` that will be used as the root module for the standard Webpack bundle. This file does not contain the interactive requires.